packages/snippets/pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: snippets
 description: A package for parsing and manipulating code samples in Flutter repo dartdoc comments.
 repository: https://github.com/flutter/assets-for-api-docs/tree/master/packages/snippets
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+snippets%22
-version: 0.3.0
+version: 0.3.1
 
 environment:
-  sdk: ">=2.17.0-0 <3.0.0"
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
   analyzer: 4.3.1
```

Comment:
Thanks!

Reply:
I think you meant the newer version of `file`  (not `file: ^6.0.0-nullsafety.4`) since the current version is already higher.

https://github.com/flutter/assets-for-api-docs/blob/0b2ee756f13c4be6923045f5dbf7cc27e973cd9d/packages/snippets/pubspec.yaml#L14